OCAPP/Controllers/Institutions.py

```diff
@@ -4,3 +4,12 @@
 @app.route('/institutions', methods=['POST'])
 def create(id):
```

Comment:
lets utilize this /institutions route for the final creation route.  I'd like to keep it as RESTful as possible.